bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/inbox/InboxFilterCriteria.java

```diff
@@ -26,61 +27,137 @@
     private final String bindingId;
     private final ThingTypeUID thingTypeUID;
     private final ThingUID thingUID;
+    private final String representationValue;
     private final DiscoveryResultFlag flag;
 
+    private InboxFilterCriteria(String bindingId, ThingTypeUID thingTypeUID, ThingUID thingUID,
```

Comment:
I could, but as @sjka suggested to make them all deprecated and to be removed before creating release 1.0 I didn't touch them. they are there for ages. If you insist, I will change them.